From c708d2811e471c0aeb3454ec864fc22d447092e7 Mon Sep 17 00:00:00 2001
From: Ding Yunliang <yunliang.ding@intel.com>
Date: Fri, 11 May 2018 10:17:45 +0800
Subject: [PATCH 112/129] media: intel-ipu4: update SPDX license

Message for Open Source:
The SPDX license format which were used before was not correct,
update them accrording to the community rule.

Message for Internal:
[Issue] Format not correct.
[Changes] Change _ to - as required.

Change-Id: Ic42115dce25aafc60893ba028c670b7f2b948e50
Tracked-On: #JICCCN-228
Signed-off-by: Ding Yunliang <yunliang.ding@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/ad5816g.c                                   | 2 +-
 drivers/media/i2c/ad5816g.h                                   | 2 +-
 drivers/media/i2c/ak7375.c                                    | 2 +-
 drivers/media/i2c/as3638.c                                    | 2 +-
 drivers/media/i2c/bu64295.c                                   | 2 +-
 drivers/media/i2c/bu64295.h                                   | 2 +-
 drivers/media/i2c/crlmodule/crl_adv7481_configuration.h       | 2 +-
 drivers/media/i2c/crlmodule/crl_adv7481_cvbs_configuration.h  | 2 +-
 drivers/media/i2c/crlmodule/crl_adv7481_eval_configuration.h  | 2 +-
 drivers/media/i2c/crlmodule/crl_adv7481_hdmi_configuration.c  | 2 +-
 drivers/media/i2c/crlmodule/crl_adv7481_hdmi_configuration.h  | 2 +-
 drivers/media/i2c/crlmodule/crl_imx132_configuration.h        | 2 +-
 drivers/media/i2c/crlmodule/crl_imx135_configuration.h        | 2 +-
 drivers/media/i2c/crlmodule/crl_imx185_configuration.h        | 2 +-
 drivers/media/i2c/crlmodule/crl_imx214_configuration.h        | 2 +-
 drivers/media/i2c/crlmodule/crl_imx230_configuration.h        | 2 +-
 drivers/media/i2c/crlmodule/crl_imx274_configuration.h        | 2 +-
 drivers/media/i2c/crlmodule/crl_imx290_configuration.h        | 2 +-
 drivers/media/i2c/crlmodule/crl_imx318_configuration.h        | 2 +-
 drivers/media/i2c/crlmodule/crl_imx477_common_regs.h          | 2 +-
 drivers/media/i2c/crlmodule/crl_imx477_master_configuration.h | 2 +-
 drivers/media/i2c/crlmodule/crl_imx477_slave_configuration.h  | 2 +-
 drivers/media/i2c/crlmodule/crl_magna_configuration.h         | 2 +-
 drivers/media/i2c/crlmodule/crl_ov10635_configuration.h       | 2 +-
 drivers/media/i2c/crlmodule/crl_ov10640_configuration.h       | 2 +-
 drivers/media/i2c/crlmodule/crl_ov13860_configuration.h       | 2 +-
 drivers/media/i2c/crlmodule/crl_ov2740_configuration.h        | 2 +-
 drivers/media/i2c/crlmodule/crl_ov5670_configuration.h        | 2 +-
 drivers/media/i2c/crlmodule/crl_ov8858_configuration.h        | 2 +-
 drivers/media/i2c/crlmodule/crl_ov9281_configuration.h        | 2 +-
 drivers/media/i2c/crlmodule/crl_pixter_stub_configuration.h   | 2 +-
 drivers/media/i2c/crlmodule/crlmodule-core.c                  | 2 +-
 drivers/media/i2c/crlmodule/crlmodule-data.c                  | 2 +-
 drivers/media/i2c/crlmodule/crlmodule-msrlist.c               | 2 +-
 drivers/media/i2c/crlmodule/crlmodule-msrlist.h               | 2 +-
 drivers/media/i2c/crlmodule/crlmodule-nvm.c                   | 2 +-
 drivers/media/i2c/crlmodule/crlmodule-nvm.h                   | 2 +-
 drivers/media/i2c/crlmodule/crlmodule-regs.c                  | 2 +-
 drivers/media/i2c/crlmodule/crlmodule-regs.h                  | 2 +-
 drivers/media/i2c/crlmodule/crlmodule-sensor-ds.h             | 2 +-
 drivers/media/i2c/crlmodule/crlmodule.h                       | 2 +-
 drivers/media/i2c/dw9714.c                                    | 2 +-
 drivers/media/i2c/imx319.c                                    | 2 +-
 drivers/media/i2c/imx355.c                                    | 2 +-
 drivers/media/i2c/lc898122/lc898122-oiscmd.c                  | 2 +-
 drivers/media/i2c/lc898122/lc898122-oisfil.h                  | 2 +-
 drivers/media/i2c/lc898122/lc898122-oisinit.c                 | 2 +-
 drivers/media/i2c/lc898122/lc898122-oisinit.h                 | 2 +-
 drivers/media/i2c/lc898122/lc898122-regs.h                    | 2 +-
 drivers/media/i2c/lc898122/lc898122.c                         | 2 +-
 drivers/media/i2c/lc898122/lc898122.h                         | 2 +-
 drivers/media/i2c/lm3643.c                                    | 2 +-
 drivers/media/i2c/ov8856.c                                    | 2 +-
 drivers/media/i2c/ti964-reg.h                                 | 2 +-
 drivers/media/i2c/ti964.c                                     | 2 +-
 drivers/media/i2c/vcm_stub.c                                  | 2 +-
 drivers/media/i2c/vcm_stub.h                                  | 2 +-
 drivers/media/pci/intel/ipu-bus.c                             | 2 +-
 drivers/media/pci/intel/ipu-bus.h                             | 2 +-
 drivers/media/pci/intel/ipu-buttress.c                        | 2 +-
 drivers/media/pci/intel/ipu-buttress.h                        | 2 +-
 drivers/media/pci/intel/ipu-cpd.c                             | 2 +-
 drivers/media/pci/intel/ipu-cpd.h                             | 2 +-
 drivers/media/pci/intel/ipu-dma.c                             | 2 +-
 drivers/media/pci/intel/ipu-dma.h                             | 2 +-
 drivers/media/pci/intel/ipu-fw-com.c                          | 2 +-
 drivers/media/pci/intel/ipu-fw-com.h                          | 2 +-
 drivers/media/pci/intel/ipu-fw-isys.c                         | 2 +-
 drivers/media/pci/intel/ipu-fw-isys.h                         | 2 +-
 drivers/media/pci/intel/ipu-fw-psys.c                         | 2 +-
 drivers/media/pci/intel/ipu-fw-psys.h                         | 2 +-
 drivers/media/pci/intel/ipu-isys-csi2-be-soc.c                | 2 +-
 drivers/media/pci/intel/ipu-isys-csi2-be.c                    | 2 +-
 drivers/media/pci/intel/ipu-isys-csi2-be.h                    | 2 +-
 drivers/media/pci/intel/ipu-isys-csi2.c                       | 2 +-
 drivers/media/pci/intel/ipu-isys-csi2.h                       | 2 +-
 drivers/media/pci/intel/ipu-isys-media.h                      | 2 +-
 drivers/media/pci/intel/ipu-isys-queue.c                      | 2 +-
 drivers/media/pci/intel/ipu-isys-queue.h                      | 2 +-
 drivers/media/pci/intel/ipu-isys-subdev.c                     | 2 +-
 drivers/media/pci/intel/ipu-isys-subdev.h                     | 2 +-
 drivers/media/pci/intel/ipu-isys-tpg.c                        | 2 +-
 drivers/media/pci/intel/ipu-isys-tpg.h                        | 2 +-
 drivers/media/pci/intel/ipu-isys-video.c                      | 2 +-
 drivers/media/pci/intel/ipu-isys-video.h                      | 2 +-
 drivers/media/pci/intel/ipu-isys.c                            | 2 +-
 drivers/media/pci/intel/ipu-isys.h                            | 2 +-
 drivers/media/pci/intel/ipu-mmu.c                             | 2 +-
 drivers/media/pci/intel/ipu-mmu.h                             | 2 +-
 drivers/media/pci/intel/ipu-pdata.h                           | 2 +-
 drivers/media/pci/intel/ipu-psys-compat32.c                   | 2 +-
 drivers/media/pci/intel/ipu-psys.c                            | 2 +-
 drivers/media/pci/intel/ipu-psys.h                            | 2 +-
 drivers/media/pci/intel/ipu-trace-event.h                     | 2 +-
 drivers/media/pci/intel/ipu-trace.c                           | 2 +-
 drivers/media/pci/intel/ipu-trace.h                           | 2 +-
 drivers/media/pci/intel/ipu-wrapper.c                         | 2 +-
 drivers/media/pci/intel/ipu-wrapper.h                         | 2 +-
 drivers/media/pci/intel/ipu.c                                 | 2 +-
 drivers/media/pci/intel/ipu.h                                 | 2 +-
 drivers/media/pci/intel/ipu4/ipu-platform-buttress-regs.h     | 2 +-
 drivers/media/pci/intel/ipu4/ipu-platform-isys-csi2-reg.h     | 2 +-
 drivers/media/pci/intel/ipu4/ipu-platform-isys.h              | 2 +-
 drivers/media/pci/intel/ipu4/ipu-platform-regs.h              | 2 +-
 drivers/media/pci/intel/ipu4/ipu-platform-resources.h         | 2 +-
 drivers/media/pci/intel/ipu4/ipu-platform.h                   | 2 +-
 drivers/media/pci/intel/ipu4/ipu4-css/libintel-ipu4.c         | 2 +-
 drivers/media/pci/intel/ipu4/ipu4-fw-resources.c              | 2 +-
 drivers/media/pci/intel/ipu4/ipu4-isys-csi2.c                 | 2 +-
 drivers/media/pci/intel/ipu4/ipu4-isys-isa.c                  | 2 +-
 drivers/media/pci/intel/ipu4/ipu4-isys-isa.h                  | 2 +-
 drivers/media/pci/intel/ipu4/ipu4-isys.c                      | 2 +-
 drivers/media/pci/intel/ipu4/ipu4-psys.c                      | 2 +-
 drivers/media/pci/intel/ipu4/ipu4-resources.c                 | 2 +-
 drivers/media/pci/intel/ipu4/ipu4.c                           | 2 +-
 drivers/media/pci/intel/ipu4/ipu4p-css/libintel-ipu4p.c       | 2 +-
 drivers/media/pci/intel/ipu4/ipu4p-isys-csi2.c                | 2 +-
 drivers/media/platform/intel/ipu4-bxt-p-pdata.c               | 2 +-
 drivers/media/platform/intel/ipu4p-cnl-rvp-pdata.c            | 2 +-
 include/media/as3638.h                                        | 2 +-
 include/media/crlmodule.h                                     | 2 +-
 include/media/dw9714.h                                        | 2 +-
 include/media/ipu-isys.h                                      | 2 +-
 include/media/lc898122.h                                      | 2 +-
 include/media/lm3643.h                                        | 2 +-
 include/media/ti964.h                                         | 2 +-
 include/uapi/linux/crlmodule.h                                | 2 +-
 include/uapi/linux/ipu-isys-isa-fw.h                          | 2 +-
 include/uapi/linux/ipu-isys.h                                 | 2 +-
 include/uapi/linux/ipu-psys.h                                 | 2 +-
 130 files changed, 130 insertions(+), 130 deletions(-)

diff --git a/drivers/media/i2c/ad5816g.c b/drivers/media/i2c/ad5816g.c
index 8f04b4d..a1d23f2 100644
--- a/drivers/media/i2c/ad5816g.c
+++ b/drivers/media/i2c/ad5816g.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2016 - 2018 Intel Corporation
 
 #include <linux/delay.h>
diff --git a/drivers/media/i2c/ad5816g.h b/drivers/media/i2c/ad5816g.h
index 14dbcbf..9c417a8 100644
--- a/drivers/media/i2c/ad5816g.h
+++ b/drivers/media/i2c/ad5816g.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation */
 #ifndef __AD5816G_H__
 #define __AD5816G_H__
diff --git a/drivers/media/i2c/ak7375.c b/drivers/media/i2c/ak7375.c
index 890983e..9150f77 100644
--- a/drivers/media/i2c/ak7375.c
+++ b/drivers/media/i2c/ak7375.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2018 Intel Corporation
 
 #include <linux/acpi.h>
diff --git a/drivers/media/i2c/as3638.c b/drivers/media/i2c/as3638.c
index d873c02..7ccdab9 100644
--- a/drivers/media/i2c/as3638.c
+++ b/drivers/media/i2c/as3638.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2016 - 2018 Intel Corporation
 
 #include <linux/module.h>
diff --git a/drivers/media/i2c/bu64295.c b/drivers/media/i2c/bu64295.c
index f8e3aa2..6d1c4c0 100644
--- a/drivers/media/i2c/bu64295.c
+++ b/drivers/media/i2c/bu64295.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2016 - 2018 Intel Corporation
 
 #include <linux/gpio.h>
diff --git a/drivers/media/i2c/bu64295.h b/drivers/media/i2c/bu64295.h
index 15c720d..36b873d 100644
--- a/drivers/media/i2c/bu64295.h
+++ b/drivers/media/i2c/bu64295.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation */
 #ifndef __BU64295_H__
 #define __BU64295_H__
diff --git a/drivers/media/i2c/crlmodule/crl_adv7481_configuration.h b/drivers/media/i2c/crlmodule/crl_adv7481_configuration.h
index 472acc0..9cf6e37 100644
--- a/drivers/media/i2c/crlmodule/crl_adv7481_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_adv7481_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2015 - 2018 Intel Corporation
  *
  * Author: Jianxu Zheng <jian.xu.zheng@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_adv7481_cvbs_configuration.h b/drivers/media/i2c/crlmodule/crl_adv7481_cvbs_configuration.h
index 39d0f88..34f33ac 100644
--- a/drivers/media/i2c/crlmodule/crl_adv7481_cvbs_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_adv7481_cvbs_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2015 - 2018 Intel Corporation
  *
  * Author: Jianxu Zheng <jian.xu.zheng@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_adv7481_eval_configuration.h b/drivers/media/i2c/crlmodule/crl_adv7481_eval_configuration.h
index b23ff35..c778170 100644
--- a/drivers/media/i2c/crlmodule/crl_adv7481_eval_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_adv7481_eval_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2015 - 2018 Intel Corporation
  *
  * Author: Jianxu Zheng <jian.xu.zheng@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_adv7481_hdmi_configuration.c b/drivers/media/i2c/crlmodule/crl_adv7481_hdmi_configuration.c
index 96465e1..d8d9c7b 100644
--- a/drivers/media/i2c/crlmodule/crl_adv7481_hdmi_configuration.c
+++ b/drivers/media/i2c/crlmodule/crl_adv7481_hdmi_configuration.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2016 - 2018 Intel Corporation
 
 #include <linux/device.h>
diff --git a/drivers/media/i2c/crlmodule/crl_adv7481_hdmi_configuration.h b/drivers/media/i2c/crlmodule/crl_adv7481_hdmi_configuration.h
index cc70c36..9a7eca4 100644
--- a/drivers/media/i2c/crlmodule/crl_adv7481_hdmi_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_adv7481_hdmi_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation */
 
 #ifndef __CRLMODULE_ADV7481_HDMI_CONFIGURATION_H_
diff --git a/drivers/media/i2c/crlmodule/crl_imx132_configuration.h b/drivers/media/i2c/crlmodule/crl_imx132_configuration.h
index 6cae84e..128ccb5 100644
--- a/drivers/media/i2c/crlmodule/crl_imx132_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_imx132_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2015 - 2018 Intel Corporation */
 
 #ifndef __CRLMODULE_IMX132_CONFIGURATION_H_
diff --git a/drivers/media/i2c/crlmodule/crl_imx135_configuration.h b/drivers/media/i2c/crlmodule/crl_imx135_configuration.h
index 135c9a1..82efac8 100644
--- a/drivers/media/i2c/crlmodule/crl_imx135_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_imx135_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2015 - 2018 Intel Corporation */
 
 #ifndef __CRLMODULE_IMX135_CONFIGURATION_H_
diff --git a/drivers/media/i2c/crlmodule/crl_imx185_configuration.h b/drivers/media/i2c/crlmodule/crl_imx185_configuration.h
index df9e480..168455b 100644
--- a/drivers/media/i2c/crlmodule/crl_imx185_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_imx185_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2017 - 2018 Intel Corporation
  *
  * Author: Shuguang Gong <shuguang.gong@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_imx214_configuration.h b/drivers/media/i2c/crlmodule/crl_imx214_configuration.h
index 0835e25..8dc536f 100644
--- a/drivers/media/i2c/crlmodule/crl_imx214_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_imx214_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2014 - 2018 Intel Corporation
  *
  * Author: Vinod Govindapillai <vinod.govindapillai@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_imx230_configuration.h b/drivers/media/i2c/crlmodule/crl_imx230_configuration.h
index af569e6..c19afad 100644
--- a/drivers/media/i2c/crlmodule/crl_imx230_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_imx230_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2014 - 2018 Intel Corporation
  *
  * Author: Vinod Govindapillai <vinod.govindapillai@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_imx274_configuration.h b/drivers/media/i2c/crlmodule/crl_imx274_configuration.h
index 558faea..6ec84fb 100644
--- a/drivers/media/i2c/crlmodule/crl_imx274_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_imx274_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation
  *
  * Author: Yuning Pu <yuning.pu@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_imx290_configuration.h b/drivers/media/i2c/crlmodule/crl_imx290_configuration.h
index 998b3d9..6a3561b 100644
--- a/drivers/media/i2c/crlmodule/crl_imx290_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_imx290_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2017 - 2018 Intel Corporation
  *
  * Author: Yuning Pu <yuning.pu@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_imx318_configuration.h b/drivers/media/i2c/crlmodule/crl_imx318_configuration.h
index 4555313..631a6d1 100644
--- a/drivers/media/i2c/crlmodule/crl_imx318_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_imx318_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation
  *
  * Author: Jouni Ukkonen <jouni.ukkonen@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_imx477_common_regs.h b/drivers/media/i2c/crlmodule/crl_imx477_common_regs.h
index ecf20b9..eebb884 100644
--- a/drivers/media/i2c/crlmodule/crl_imx477_common_regs.h
+++ b/drivers/media/i2c/crlmodule/crl_imx477_common_regs.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2017 - 2018 Intel Corporation
  *
  * Author: Alexei Zavjalov <alexei.zavjalov@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_imx477_master_configuration.h b/drivers/media/i2c/crlmodule/crl_imx477_master_configuration.h
index f28343c..10be93b 100644
--- a/drivers/media/i2c/crlmodule/crl_imx477_master_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_imx477_master_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2017 - 2018 Intel Corporation
  *
  * Author: Alexei Zavjalov <alexei.zavjalov@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_imx477_slave_configuration.h b/drivers/media/i2c/crlmodule/crl_imx477_slave_configuration.h
index 9e9f040..b8dc15c 100644
--- a/drivers/media/i2c/crlmodule/crl_imx477_slave_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_imx477_slave_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2017 - 2018 Intel Corporation
  *
  * Author: Alexei Zavjalov <alexei.zavjalov@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_magna_configuration.h b/drivers/media/i2c/crlmodule/crl_magna_configuration.h
index bb2dee1..cd1e316 100644
--- a/drivers/media/i2c/crlmodule/crl_magna_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_magna_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2017 - 2018 Intel Corporation
  *
  * Author: Kishore Bodke <kishore.k.bodke@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_ov10635_configuration.h b/drivers/media/i2c/crlmodule/crl_ov10635_configuration.h
index c056e4b..d771953 100644
--- a/drivers/media/i2c/crlmodule/crl_ov10635_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_ov10635_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation
  *
  * Author: Yunliang Ding <yunliang.ding@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_ov10640_configuration.h b/drivers/media/i2c/crlmodule/crl_ov10640_configuration.h
index da3d332..ab8378b 100644
--- a/drivers/media/i2c/crlmodule/crl_ov10640_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_ov10640_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2017 - 2018 Intel Corporation
  *
  * Author: Shuguang Gong <shuguang.gong@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_ov13860_configuration.h b/drivers/media/i2c/crlmodule/crl_ov13860_configuration.h
index 2c69b93..4f8e7c6 100644
--- a/drivers/media/i2c/crlmodule/crl_ov13860_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_ov13860_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2015 - 2018 Intel Corporation
  *
  * Author: Kamal Ramamoorthy <kamalanathan.ramamoorthy@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_ov2740_configuration.h b/drivers/media/i2c/crlmodule/crl_ov2740_configuration.h
index 58a3f67..f6c56f3 100755
--- a/drivers/media/i2c/crlmodule/crl_ov2740_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_ov2740_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2017 - 2018 Intel Corporation
  *
  * Author: Roy Yang <royx.yang@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_ov5670_configuration.h b/drivers/media/i2c/crlmodule/crl_ov5670_configuration.h
index d1579dd..7badb60 100644
--- a/drivers/media/i2c/crlmodule/crl_ov5670_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_ov5670_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation
  *
  * Author: Tommi Franttila <tommi.franttila@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_ov8858_configuration.h b/drivers/media/i2c/crlmodule/crl_ov8858_configuration.h
index e2c706e..63faedc 100644
--- a/drivers/media/i2c/crlmodule/crl_ov8858_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_ov8858_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2014 - 2018 Intel Corporation
  *
  * Author: Vinod Govindapillai <vinod.govindapillai@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_ov9281_configuration.h b/drivers/media/i2c/crlmodule/crl_ov9281_configuration.h
index 75fe0ba..b052d30 100644
--- a/drivers/media/i2c/crlmodule/crl_ov9281_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_ov9281_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2018 Intel Corporation
  *
  * Author: Wu Xia <xia.wu@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crl_pixter_stub_configuration.h b/drivers/media/i2c/crlmodule/crl_pixter_stub_configuration.h
index d33eb18..e6ffbb2 100644
--- a/drivers/media/i2c/crlmodule/crl_pixter_stub_configuration.h
+++ b/drivers/media/i2c/crlmodule/crl_pixter_stub_configuration.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation
  *
  * Author: Wang, Zaikuo <zaikuo.wang@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crlmodule-core.c b/drivers/media/i2c/crlmodule/crlmodule-core.c
index faeedf8..a4cd4bc 100755
--- a/drivers/media/i2c/crlmodule/crlmodule-core.c
+++ b/drivers/media/i2c/crlmodule/crlmodule-core.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 /* Copyright (C) 2014 - 2018 Intel Corporation
  *
  * Author: Vinod Govindapillai <vinod.govindapillai@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crlmodule-data.c b/drivers/media/i2c/crlmodule/crlmodule-data.c
index 512c597..b1a2412 100755
--- a/drivers/media/i2c/crlmodule/crlmodule-data.c
+++ b/drivers/media/i2c/crlmodule/crlmodule-data.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 /* Copyright (C) 2014 - 2018 Intel Corporation
  *
  * Author: Vinod Govindapillai <vinod.govindapillai@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crlmodule-msrlist.c b/drivers/media/i2c/crlmodule/crlmodule-msrlist.c
index 732f6c1..a15b76b 100644
--- a/drivers/media/i2c/crlmodule/crlmodule-msrlist.c
+++ b/drivers/media/i2c/crlmodule/crlmodule-msrlist.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2016 - 2018 Intel Corporation
 
 #include <linux/i2c.h>
diff --git a/drivers/media/i2c/crlmodule/crlmodule-msrlist.h b/drivers/media/i2c/crlmodule/crlmodule-msrlist.h
index 978ec02..013469b 100644
--- a/drivers/media/i2c/crlmodule/crlmodule-msrlist.h
+++ b/drivers/media/i2c/crlmodule/crlmodule-msrlist.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation */
 
 #ifndef __CRLMODULE_MSRLIST_H__
diff --git a/drivers/media/i2c/crlmodule/crlmodule-nvm.c b/drivers/media/i2c/crlmodule/crlmodule-nvm.c
index ef85b0c..691027b6 100755
--- a/drivers/media/i2c/crlmodule/crlmodule-nvm.c
+++ b/drivers/media/i2c/crlmodule/crlmodule-nvm.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 /* Copyright (C) 2016 - 2018 Intel Corporation
  *
  * Author: Tommi Franttila <tommi.franttila@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crlmodule-nvm.h b/drivers/media/i2c/crlmodule/crlmodule-nvm.h
index 63093d0..42d462d 100644
--- a/drivers/media/i2c/crlmodule/crlmodule-nvm.h
+++ b/drivers/media/i2c/crlmodule/crlmodule-nvm.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2015 - 2018 Intel Corporation
  *
  * Author: Tommi Franttila <tommi.franttila@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crlmodule-regs.c b/drivers/media/i2c/crlmodule/crlmodule-regs.c
index 5f09b3d..e4b8c8a 100644
--- a/drivers/media/i2c/crlmodule/crlmodule-regs.c
+++ b/drivers/media/i2c/crlmodule/crlmodule-regs.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 /* Copyright (C) 2017 - 2018 Intel Corporation
  *
  * Author: Vinod Govindapillai <vinod.govindapillai@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crlmodule-regs.h b/drivers/media/i2c/crlmodule/crlmodule-regs.h
index dc2c7d7..6d84486 100644
--- a/drivers/media/i2c/crlmodule/crlmodule-regs.h
+++ b/drivers/media/i2c/crlmodule/crlmodule-regs.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2014 - 2018 Intel Corporation
  *
  * Author: Vinod Govindapillai <vinod.govindapillai@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crlmodule-sensor-ds.h b/drivers/media/i2c/crlmodule/crlmodule-sensor-ds.h
index fdd2c47..ff03185 100644
--- a/drivers/media/i2c/crlmodule/crlmodule-sensor-ds.h
+++ b/drivers/media/i2c/crlmodule/crlmodule-sensor-ds.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2014 - 2018 Intel Corporation
  *
  * Author: Vinod Govindapillai <vinod.govindapillai@intel.com>
diff --git a/drivers/media/i2c/crlmodule/crlmodule.h b/drivers/media/i2c/crlmodule/crlmodule.h
index 3791ccd..e68e82f 100644
--- a/drivers/media/i2c/crlmodule/crlmodule.h
+++ b/drivers/media/i2c/crlmodule/crlmodule.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2014 - 2018 Intel Corporation
  *
  * Author: Vinod Govindapillai <vinod.govindapillai@intel.com>
diff --git a/drivers/media/i2c/dw9714.c b/drivers/media/i2c/dw9714.c
index 4785b2e..da7b4f1 100644
--- a/drivers/media/i2c/dw9714.c
+++ b/drivers/media/i2c/dw9714.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 /* Copyright (C) 2015 - 2018 Intel Corporation
  *
  * Based on ATOMISP dw9714 implementation by
diff --git a/drivers/media/i2c/imx319.c b/drivers/media/i2c/imx319.c
index bb84981..9fee398 100644
--- a/drivers/media/i2c/imx319.c
+++ b/drivers/media/i2c/imx319.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2018 Intel Corporation
 
 #include <asm/unaligned.h>
diff --git a/drivers/media/i2c/imx355.c b/drivers/media/i2c/imx355.c
index 27b5cfe..3c092d0 100644
--- a/drivers/media/i2c/imx355.c
+++ b/drivers/media/i2c/imx355.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2017 - 2018 Intel Corporation
 
 #include <asm/unaligned.h>
diff --git a/drivers/media/i2c/lc898122/lc898122-oiscmd.c b/drivers/media/i2c/lc898122/lc898122-oiscmd.c
index fd5d545..4bb4b26 100644
--- a/drivers/media/i2c/lc898122/lc898122-oiscmd.c
+++ b/drivers/media/i2c/lc898122/lc898122-oiscmd.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 /* Copyright (C) 2015 - 2018 Intel Corporation
  * Copyright (C) ON Semiconductor
  *
diff --git a/drivers/media/i2c/lc898122/lc898122-oisfil.h b/drivers/media/i2c/lc898122/lc898122-oisfil.h
index db6374d..5e73e50 100644
--- a/drivers/media/i2c/lc898122/lc898122-oisfil.h
+++ b/drivers/media/i2c/lc898122/lc898122-oisfil.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2015 - 2018 Intel Corporation
  * Copyright (C) ON Semiconductor
  *
diff --git a/drivers/media/i2c/lc898122/lc898122-oisinit.c b/drivers/media/i2c/lc898122/lc898122-oisinit.c
index b5895ee..429eb70 100644
--- a/drivers/media/i2c/lc898122/lc898122-oisinit.c
+++ b/drivers/media/i2c/lc898122/lc898122-oisinit.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 /* Copyright (C) 2015 - 2018 Intel Corporation
  * Copyright (C) ON Semiconductor
  *
diff --git a/drivers/media/i2c/lc898122/lc898122-oisinit.h b/drivers/media/i2c/lc898122/lc898122-oisinit.h
index 1c79040..d619fe0 100644
--- a/drivers/media/i2c/lc898122/lc898122-oisinit.h
+++ b/drivers/media/i2c/lc898122/lc898122-oisinit.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2015 - 2018 Intel Corporation
  * Copyright (C) ON Semiconductor
  *
diff --git a/drivers/media/i2c/lc898122/lc898122-regs.h b/drivers/media/i2c/lc898122/lc898122-regs.h
index a6eea9e..74caa384 100644
--- a/drivers/media/i2c/lc898122/lc898122-regs.h
+++ b/drivers/media/i2c/lc898122/lc898122-regs.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2015 - 2018 Intel Corporation
  * Copyright (C) ON Semiconductor
  *
diff --git a/drivers/media/i2c/lc898122/lc898122.c b/drivers/media/i2c/lc898122/lc898122.c
index 9eda99a..41cff00 100644
--- a/drivers/media/i2c/lc898122/lc898122.c
+++ b/drivers/media/i2c/lc898122/lc898122.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2015 - 2018 Intel Corporation
 
 #include <linux/gpio.h>
diff --git a/drivers/media/i2c/lc898122/lc898122.h b/drivers/media/i2c/lc898122/lc898122.h
index 2704a6d..5edf017 100644
--- a/drivers/media/i2c/lc898122/lc898122.h
+++ b/drivers/media/i2c/lc898122/lc898122.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2015 - 2018 Intel Corporation */
 
 #ifndef __LC898122_H__
diff --git a/drivers/media/i2c/lm3643.c b/drivers/media/i2c/lm3643.c
index a7c44e6..695a238 100644
--- a/drivers/media/i2c/lm3643.c
+++ b/drivers/media/i2c/lm3643.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 /* Copyright (C) 2014 - 2018 Intel Corporation
  *
  * Lm3560 and other TI drivers used for this, written by
diff --git a/drivers/media/i2c/ov8856.c b/drivers/media/i2c/ov8856.c
index 16d253f..9c8fabd0 100644
--- a/drivers/media/i2c/ov8856.c
+++ b/drivers/media/i2c/ov8856.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2017 - 2018 Intel Corporation
 
 #include <linux/acpi.h>
diff --git a/drivers/media/i2c/ti964-reg.h b/drivers/media/i2c/ti964-reg.h
index 6800a1a..e916c41 100644
--- a/drivers/media/i2c/ti964-reg.h
+++ b/drivers/media/i2c/ti964-reg.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation */
 
 #ifndef TI964_REG_H
diff --git a/drivers/media/i2c/ti964.c b/drivers/media/i2c/ti964.c
index 203f7f0..7f97cd4 100644
--- a/drivers/media/i2c/ti964.c
+++ b/drivers/media/i2c/ti964.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2016 - 2018 Intel Corporation
 
 #include <linux/device.h>
diff --git a/drivers/media/i2c/vcm_stub.c b/drivers/media/i2c/vcm_stub.c
index 33d7565..773c2a3 100644
--- a/drivers/media/i2c/vcm_stub.c
+++ b/drivers/media/i2c/vcm_stub.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2017 - 2018 Intel Corporation
 
 #include <linux/i2c.h>
diff --git a/drivers/media/i2c/vcm_stub.h b/drivers/media/i2c/vcm_stub.h
index dd7d0bd..f609ed2 100644
--- a/drivers/media/i2c/vcm_stub.h
+++ b/drivers/media/i2c/vcm_stub.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2017 - 2018 Intel Corporation */
 
 #ifndef _VCM_STUB_H_
diff --git a/drivers/media/pci/intel/ipu-bus.c b/drivers/media/pci/intel/ipu-bus.c
index 886f28c..a50b6ab 100644
--- a/drivers/media/pci/intel/ipu-bus.c
+++ b/drivers/media/pci/intel/ipu-bus.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <linux/delay.h>
diff --git a/drivers/media/pci/intel/ipu-bus.h b/drivers/media/pci/intel/ipu-bus.h
index 751e93d..4226b86 100644
--- a/drivers/media/pci/intel/ipu-bus.h
+++ b/drivers/media/pci/intel/ipu-bus.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_BUS_H
diff --git a/drivers/media/pci/intel/ipu-buttress.c b/drivers/media/pci/intel/ipu-buttress.c
index ab40787..fc81338 100644
--- a/drivers/media/pci/intel/ipu-buttress.c
+++ b/drivers/media/pci/intel/ipu-buttress.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <linux/clk.h>
diff --git a/drivers/media/pci/intel/ipu-buttress.h b/drivers/media/pci/intel/ipu-buttress.h
index fcb9aae..7c29db5 100644
--- a/drivers/media/pci/intel/ipu-buttress.h
+++ b/drivers/media/pci/intel/ipu-buttress.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_BUTTRESS_H
diff --git a/drivers/media/pci/intel/ipu-cpd.c b/drivers/media/pci/intel/ipu-cpd.c
index 602ca15..dca0323 100644
--- a/drivers/media/pci/intel/ipu-cpd.c
+++ b/drivers/media/pci/intel/ipu-cpd.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2015 - 2018 Intel Corporation
 
 #include <linux/dma-mapping.h>
diff --git a/drivers/media/pci/intel/ipu-cpd.h b/drivers/media/pci/intel/ipu-cpd.h
index 47877f1..c91ae33 100644
--- a/drivers/media/pci/intel/ipu-cpd.h
+++ b/drivers/media/pci/intel/ipu-cpd.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2015 - 2018 Intel Corporation */
 
 #ifndef IPU_CPD_H
diff --git a/drivers/media/pci/intel/ipu-dma.c b/drivers/media/pci/intel/ipu-dma.c
index 2d27780..0770d8d 100644
--- a/drivers/media/pci/intel/ipu-dma.c
+++ b/drivers/media/pci/intel/ipu-dma.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <asm/cacheflush.h>
diff --git a/drivers/media/pci/intel/ipu-dma.h b/drivers/media/pci/intel/ipu-dma.h
index d20c034..9974b69 100644
--- a/drivers/media/pci/intel/ipu-dma.h
+++ b/drivers/media/pci/intel/ipu-dma.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_DMA_H
diff --git a/drivers/media/pci/intel/ipu-fw-com.c b/drivers/media/pci/intel/ipu-fw-com.c
index 839fb92..78c5c92 100644
--- a/drivers/media/pci/intel/ipu-fw-com.c
+++ b/drivers/media/pci/intel/ipu-fw-com.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <asm/cacheflush.h>
diff --git a/drivers/media/pci/intel/ipu-fw-com.h b/drivers/media/pci/intel/ipu-fw-com.h
index 9a39ac5..de47455 100644
--- a/drivers/media/pci/intel/ipu-fw-com.h
+++ b/drivers/media/pci/intel/ipu-fw-com.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_FW_COM_H
diff --git a/drivers/media/pci/intel/ipu-fw-isys.c b/drivers/media/pci/intel/ipu-fw-isys.c
index 6fc2917..ca0bdba 100644
--- a/drivers/media/pci/intel/ipu-fw-isys.c
+++ b/drivers/media/pci/intel/ipu-fw-isys.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <asm/cacheflush.h>
diff --git a/drivers/media/pci/intel/ipu-fw-isys.h b/drivers/media/pci/intel/ipu-fw-isys.h
index 8ef68d4..1106202 100644
--- a/drivers/media/pci/intel/ipu-fw-isys.h
+++ b/drivers/media/pci/intel/ipu-fw-isys.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_FW_ISYS_H
diff --git a/drivers/media/pci/intel/ipu-fw-psys.c b/drivers/media/pci/intel/ipu-fw-psys.c
index a8626bc..5f0f601 100644
--- a/drivers/media/pci/intel/ipu-fw-psys.c
+++ b/drivers/media/pci/intel/ipu-fw-psys.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2016 - 2018 Intel Corporation
 
 #include <linux/delay.h>
diff --git a/drivers/media/pci/intel/ipu-fw-psys.h b/drivers/media/pci/intel/ipu-fw-psys.h
index 58c2bce..187a20a 100644
--- a/drivers/media/pci/intel/ipu-fw-psys.h
+++ b/drivers/media/pci/intel/ipu-fw-psys.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation */
 
 #ifndef IPU_FW_PSYS_H
diff --git a/drivers/media/pci/intel/ipu-isys-csi2-be-soc.c b/drivers/media/pci/intel/ipu-isys-csi2-be-soc.c
index b0f9739..8648b5f 100644
--- a/drivers/media/pci/intel/ipu-isys-csi2-be-soc.c
+++ b/drivers/media/pci/intel/ipu-isys-csi2-be-soc.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2014 - 2018 Intel Corporation
 
 #include <linux/device.h>
diff --git a/drivers/media/pci/intel/ipu-isys-csi2-be.c b/drivers/media/pci/intel/ipu-isys-csi2-be.c
index 02ce561..deaf2a5 100644
--- a/drivers/media/pci/intel/ipu-isys-csi2-be.c
+++ b/drivers/media/pci/intel/ipu-isys-csi2-be.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2014 - 2018 Intel Corporation
 
 #include <linux/device.h>
diff --git a/drivers/media/pci/intel/ipu-isys-csi2-be.h b/drivers/media/pci/intel/ipu-isys-csi2-be.h
index d65926a..70a1783 100644
--- a/drivers/media/pci/intel/ipu-isys-csi2-be.h
+++ b/drivers/media/pci/intel/ipu-isys-csi2-be.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2014 - 2018 Intel Corporation */
 
 #ifndef IPU_ISYS_CSI2_BE_H
diff --git a/drivers/media/pci/intel/ipu-isys-csi2.c b/drivers/media/pci/intel/ipu-isys-csi2.c
index c10ea32..4667df0 100644
--- a/drivers/media/pci/intel/ipu-isys-csi2.c
+++ b/drivers/media/pci/intel/ipu-isys-csi2.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <linux/device.h>
diff --git a/drivers/media/pci/intel/ipu-isys-csi2.h b/drivers/media/pci/intel/ipu-isys-csi2.h
index 10836c3..a464df9 100644
--- a/drivers/media/pci/intel/ipu-isys-csi2.h
+++ b/drivers/media/pci/intel/ipu-isys-csi2.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_ISYS_CSI2_H
diff --git a/drivers/media/pci/intel/ipu-isys-media.h b/drivers/media/pci/intel/ipu-isys-media.h
index 1d88e87..823324e 100644
--- a/drivers/media/pci/intel/ipu-isys-media.h
+++ b/drivers/media/pci/intel/ipu-isys-media.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation */
 
 #ifndef IPU_ISYS_MEDIA_H
diff --git a/drivers/media/pci/intel/ipu-isys-queue.c b/drivers/media/pci/intel/ipu-isys-queue.c
index d716e13..1f6f769 100644
--- a/drivers/media/pci/intel/ipu-isys-queue.c
+++ b/drivers/media/pci/intel/ipu-isys-queue.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <linux/completion.h>
diff --git a/drivers/media/pci/intel/ipu-isys-queue.h b/drivers/media/pci/intel/ipu-isys-queue.h
index 2a2e276..4162b05 100644
--- a/drivers/media/pci/intel/ipu-isys-queue.h
+++ b/drivers/media/pci/intel/ipu-isys-queue.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_ISYS_QUEUE_H
diff --git a/drivers/media/pci/intel/ipu-isys-subdev.c b/drivers/media/pci/intel/ipu-isys-subdev.c
index b73b532..7a72df4 100644
--- a/drivers/media/pci/intel/ipu-isys-subdev.c
+++ b/drivers/media/pci/intel/ipu-isys-subdev.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <linux/types.h>
diff --git a/drivers/media/pci/intel/ipu-isys-subdev.h b/drivers/media/pci/intel/ipu-isys-subdev.h
index 1014669..6b08cf5 100644
--- a/drivers/media/pci/intel/ipu-isys-subdev.h
+++ b/drivers/media/pci/intel/ipu-isys-subdev.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_ISYS_SUBDEV_H
diff --git a/drivers/media/pci/intel/ipu-isys-tpg.c b/drivers/media/pci/intel/ipu-isys-tpg.c
index f15e1b4..5fe0048 100644
--- a/drivers/media/pci/intel/ipu-isys-tpg.c
+++ b/drivers/media/pci/intel/ipu-isys-tpg.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <linux/device.h>
diff --git a/drivers/media/pci/intel/ipu-isys-tpg.h b/drivers/media/pci/intel/ipu-isys-tpg.h
index aae1ca6..4a2a23d 100644
--- a/drivers/media/pci/intel/ipu-isys-tpg.h
+++ b/drivers/media/pci/intel/ipu-isys-tpg.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_ISYS_TPG_H
diff --git a/drivers/media/pci/intel/ipu-isys-video.c b/drivers/media/pci/intel/ipu-isys-video.c
index 6d3edf0..0c6e59b 100644
--- a/drivers/media/pci/intel/ipu-isys-video.c
+++ b/drivers/media/pci/intel/ipu-isys-video.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <linux/delay.h>
diff --git a/drivers/media/pci/intel/ipu-isys-video.h b/drivers/media/pci/intel/ipu-isys-video.h
index c7a264b..c1375f7 100644
--- a/drivers/media/pci/intel/ipu-isys-video.h
+++ b/drivers/media/pci/intel/ipu-isys-video.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_ISYS_VIDEO_H
diff --git a/drivers/media/pci/intel/ipu-isys.c b/drivers/media/pci/intel/ipu-isys.c
index 5e4aecd..6ce6bab 100644
--- a/drivers/media/pci/intel/ipu-isys.c
+++ b/drivers/media/pci/intel/ipu-isys.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <linux/debugfs.h>
diff --git a/drivers/media/pci/intel/ipu-isys.h b/drivers/media/pci/intel/ipu-isys.h
index f735172..b362f86 100644
--- a/drivers/media/pci/intel/ipu-isys.h
+++ b/drivers/media/pci/intel/ipu-isys.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_ISYS_H
diff --git a/drivers/media/pci/intel/ipu-mmu.c b/drivers/media/pci/intel/ipu-mmu.c
index 0e40f1f..f46f479 100644
--- a/drivers/media/pci/intel/ipu-mmu.c
+++ b/drivers/media/pci/intel/ipu-mmu.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <asm/cacheflush.h>
diff --git a/drivers/media/pci/intel/ipu-mmu.h b/drivers/media/pci/intel/ipu-mmu.h
index 68c8b06..0e8863a 100644
--- a/drivers/media/pci/intel/ipu-mmu.h
+++ b/drivers/media/pci/intel/ipu-mmu.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_MMU_H
diff --git a/drivers/media/pci/intel/ipu-pdata.h b/drivers/media/pci/intel/ipu-pdata.h
index 1ca3a1c..66f1112 100644
--- a/drivers/media/pci/intel/ipu-pdata.h
+++ b/drivers/media/pci/intel/ipu-pdata.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_PDATA_H
diff --git a/drivers/media/pci/intel/ipu-psys-compat32.c b/drivers/media/pci/intel/ipu-psys-compat32.c
index ab7f94d..452fd38 100644
--- a/drivers/media/pci/intel/ipu-psys-compat32.c
+++ b/drivers/media/pci/intel/ipu-psys-compat32.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <linux/compat.h>
diff --git a/drivers/media/pci/intel/ipu-psys.c b/drivers/media/pci/intel/ipu-psys.c
index 6dc2821..714a790 100644
--- a/drivers/media/pci/intel/ipu-psys.c
+++ b/drivers/media/pci/intel/ipu-psys.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <linux/debugfs.h>
diff --git a/drivers/media/pci/intel/ipu-psys.h b/drivers/media/pci/intel/ipu-psys.h
index 27dda49..762950f 100644
--- a/drivers/media/pci/intel/ipu-psys.h
+++ b/drivers/media/pci/intel/ipu-psys.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_PSYS_H
diff --git a/drivers/media/pci/intel/ipu-trace-event.h b/drivers/media/pci/intel/ipu-trace-event.h
index 173c931..b5e8d4b 100644
--- a/drivers/media/pci/intel/ipu-trace-event.h
+++ b/drivers/media/pci/intel/ipu-trace-event.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2015 - 2018 Intel Corporation */
 
 #undef TRACE_SYSTEM
diff --git a/drivers/media/pci/intel/ipu-trace.c b/drivers/media/pci/intel/ipu-trace.c
index bece0ab..5e0795d 100644
--- a/drivers/media/pci/intel/ipu-trace.c
+++ b/drivers/media/pci/intel/ipu-trace.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2014 - 2018 Intel Corporation
 
 #include <linux/debugfs.h>
diff --git a/drivers/media/pci/intel/ipu-trace.h b/drivers/media/pci/intel/ipu-trace.h
index 4c54436..9167c04 100644
--- a/drivers/media/pci/intel/ipu-trace.h
+++ b/drivers/media/pci/intel/ipu-trace.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2014 - 2018 Intel Corporation */
 
 #ifndef IPU_TRACE_H
diff --git a/drivers/media/pci/intel/ipu-wrapper.c b/drivers/media/pci/intel/ipu-wrapper.c
index 456f82c..9e06887 100644
--- a/drivers/media/pci/intel/ipu-wrapper.c
+++ b/drivers/media/pci/intel/ipu-wrapper.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <asm/cacheflush.h>
diff --git a/drivers/media/pci/intel/ipu-wrapper.h b/drivers/media/pci/intel/ipu-wrapper.h
index 4441d05..b7df285 100644
--- a/drivers/media/pci/intel/ipu-wrapper.h
+++ b/drivers/media/pci/intel/ipu-wrapper.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_WRAPPER_H
diff --git a/drivers/media/pci/intel/ipu.c b/drivers/media/pci/intel/ipu.c
index d1f955e..cccd6d2 100644
--- a/drivers/media/pci/intel/ipu.c
+++ b/drivers/media/pci/intel/ipu.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include <linux/debugfs.h>
diff --git a/drivers/media/pci/intel/ipu.h b/drivers/media/pci/intel/ipu.h
index 3c1241e..1916018 100644
--- a/drivers/media/pci/intel/ipu.h
+++ b/drivers/media/pci/intel/ipu.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_H
diff --git a/drivers/media/pci/intel/ipu4/ipu-platform-buttress-regs.h b/drivers/media/pci/intel/ipu4/ipu-platform-buttress-regs.h
index a2684ca..ffd770c 100644
--- a/drivers/media/pci/intel/ipu4/ipu-platform-buttress-regs.h
+++ b/drivers/media/pci/intel/ipu4/ipu-platform-buttress-regs.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2014 - 2018 Intel Corporation */
 
 #ifndef IPU_PLATFORM_BUTTRESS_REGS_H
diff --git a/drivers/media/pci/intel/ipu4/ipu-platform-isys-csi2-reg.h b/drivers/media/pci/intel/ipu4/ipu-platform-isys-csi2-reg.h
index f504d9f..f19372b 100644
--- a/drivers/media/pci/intel/ipu4/ipu-platform-isys-csi2-reg.h
+++ b/drivers/media/pci/intel/ipu4/ipu-platform-isys-csi2-reg.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_PLATFORM_ISYS_CSI2_REG_H
diff --git a/drivers/media/pci/intel/ipu4/ipu-platform-isys.h b/drivers/media/pci/intel/ipu4/ipu-platform-isys.h
index 8e24406..51f26f2 100644
--- a/drivers/media/pci/intel/ipu4/ipu-platform-isys.h
+++ b/drivers/media/pci/intel/ipu4/ipu-platform-isys.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2018 Intel Corporation */
 
 #ifndef IPU_PLATFORM_ISYS_H
diff --git a/drivers/media/pci/intel/ipu4/ipu-platform-regs.h b/drivers/media/pci/intel/ipu4/ipu-platform-regs.h
index 7f27273..e54b2b5 100644
--- a/drivers/media/pci/intel/ipu4/ipu-platform-regs.h
+++ b/drivers/media/pci/intel/ipu4/ipu-platform-regs.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2014 - 2018 Intel Corporation */
 
 #ifndef IPU_PLATFORM_REGS_H
diff --git a/drivers/media/pci/intel/ipu4/ipu-platform-resources.h b/drivers/media/pci/intel/ipu4/ipu-platform-resources.h
index 52dbd1e..59b2cd4 100644
--- a/drivers/media/pci/intel/ipu4/ipu-platform-resources.h
+++ b/drivers/media/pci/intel/ipu4/ipu-platform-resources.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation */
 
 #ifndef IPU_PLATFORM_RESOURCES_H
diff --git a/drivers/media/pci/intel/ipu4/ipu-platform.h b/drivers/media/pci/intel/ipu4/ipu-platform.h
index d8af4ba..924cb2e 100644
--- a/drivers/media/pci/intel/ipu4/ipu-platform.h
+++ b/drivers/media/pci/intel/ipu4/ipu-platform.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef IPU_PLATFORM_H
diff --git a/drivers/media/pci/intel/ipu4/ipu4-css/libintel-ipu4.c b/drivers/media/pci/intel/ipu4/ipu4-css/libintel-ipu4.c
index 89234c0..a712889 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-css/libintel-ipu4.c
+++ b/drivers/media/pci/intel/ipu4/ipu4-css/libintel-ipu4.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2014 - 2018 Intel Corporation
 
 #include <linux/module.h>
diff --git a/drivers/media/pci/intel/ipu4/ipu4-fw-resources.c b/drivers/media/pci/intel/ipu4/ipu4-fw-resources.c
index 7d50bce..581b737 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-fw-resources.c
+++ b/drivers/media/pci/intel/ipu4/ipu4-fw-resources.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2015 - 2018 Intel Corporation
 
 #include "ipu-fw-psys.h"
diff --git a/drivers/media/pci/intel/ipu4/ipu4-isys-csi2.c b/drivers/media/pci/intel/ipu4/ipu4-isys-csi2.c
index 0c9392d..11e4091 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-isys-csi2.c
+++ b/drivers/media/pci/intel/ipu4/ipu4-isys-csi2.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2013 - 2018 Intel Corporation
 
 #include "ipu.h"
diff --git a/drivers/media/pci/intel/ipu4/ipu4-isys-isa.c b/drivers/media/pci/intel/ipu4/ipu4-isys-isa.c
index ff17d59..e5af62a 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-isys-isa.c
+++ b/drivers/media/pci/intel/ipu4/ipu4-isys-isa.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2014 - 2018 Intel Corporation
 
 #include <linux/device.h>
diff --git a/drivers/media/pci/intel/ipu4/ipu4-isys-isa.h b/drivers/media/pci/intel/ipu4/ipu4-isys-isa.h
index a32157f..649714d 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-isys-isa.h
+++ b/drivers/media/pci/intel/ipu4/ipu4-isys-isa.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2014 - 2018 Intel Corporation */
 
 #ifndef IPU_ISYS_ISA_H
diff --git a/drivers/media/pci/intel/ipu4/ipu4-isys.c b/drivers/media/pci/intel/ipu4/ipu4-isys.c
index 6848eba..30cc42d 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-isys.c
+++ b/drivers/media/pci/intel/ipu4/ipu4-isys.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2018 Intel Corporation
 
 #include <linux/module.h>
diff --git a/drivers/media/pci/intel/ipu4/ipu4-psys.c b/drivers/media/pci/intel/ipu4/ipu4-psys.c
index a90ce0a..82e80d1 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-psys.c
+++ b/drivers/media/pci/intel/ipu4/ipu4-psys.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2018 Intel Corporation
 
 #include <linux/uaccess.h>
diff --git a/drivers/media/pci/intel/ipu4/ipu4-resources.c b/drivers/media/pci/intel/ipu4/ipu4-resources.c
index 6510450..097ea1b 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-resources.c
+++ b/drivers/media/pci/intel/ipu4/ipu4-resources.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2015 - 2018 Intel Corporation
 
 #include <linux/bitmap.h>
diff --git a/drivers/media/pci/intel/ipu4/ipu4.c b/drivers/media/pci/intel/ipu4/ipu4.c
index 74a85eb..c3615d2 100644
--- a/drivers/media/pci/intel/ipu4/ipu4.c
+++ b/drivers/media/pci/intel/ipu4/ipu4.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2018 Intel Corporation
 
 #include <linux/device.h>
diff --git a/drivers/media/pci/intel/ipu4/ipu4p-css/libintel-ipu4p.c b/drivers/media/pci/intel/ipu4/ipu4p-css/libintel-ipu4p.c
index b6c5f1c..cb6fd04 100644
--- a/drivers/media/pci/intel/ipu4/ipu4p-css/libintel-ipu4p.c
+++ b/drivers/media/pci/intel/ipu4/ipu4p-css/libintel-ipu4p.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2014 - 2018 Intel Corporation
 
 #include <linux/module.h>
diff --git a/drivers/media/pci/intel/ipu4/ipu4p-isys-csi2.c b/drivers/media/pci/intel/ipu4/ipu4p-isys-csi2.c
index 934db9e..0ced354 100644
--- a/drivers/media/pci/intel/ipu4/ipu4p-isys-csi2.c
+++ b/drivers/media/pci/intel/ipu4/ipu4p-isys-csi2.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 //  Copyright (C) 2018 Intel Corporation
 
 #include "ipu.h"
diff --git a/drivers/media/platform/intel/ipu4-bxt-p-pdata.c b/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
index d6fab2a..53c0037 100755
--- a/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
+++ b/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2015 - 2018 Intel Corporation
 
 #include <linux/clk.h>
diff --git a/drivers/media/platform/intel/ipu4p-cnl-rvp-pdata.c b/drivers/media/platform/intel/ipu4p-cnl-rvp-pdata.c
index 633d7ad..74bfe32 100644
--- a/drivers/media/platform/intel/ipu4p-cnl-rvp-pdata.c
+++ b/drivers/media/platform/intel/ipu4p-cnl-rvp-pdata.c
@@ -1,4 +1,4 @@
-// SPDX-License_Identifier: GPL-2.0
+// SPDX-License-Identifier: GPL-2.0
 // Copyright (C) 2018 Intel Corporation
 
 #include <linux/clk.h>
diff --git a/include/media/as3638.h b/include/media/as3638.h
index 949dc6d..8e847ef 100644
--- a/include/media/as3638.h
+++ b/include/media/as3638.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation */
 
 #ifndef __AS3638_H__
diff --git a/include/media/crlmodule.h b/include/media/crlmodule.h
index 4605522..4517430 100644
--- a/include/media/crlmodule.h
+++ b/include/media/crlmodule.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2014 - 2018 Intel Corporation
  * Generic driver for common register list based camera sensor modules
  *
diff --git a/include/media/dw9714.h b/include/media/dw9714.h
index 5898629..11737c6 100755
--- a/include/media/dw9714.h
+++ b/include/media/dw9714.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2015 - 2018 Intel Corporation
  *
  * Based on ATOMISP dw9714 implementation by
diff --git a/include/media/ipu-isys.h b/include/media/ipu-isys.h
index be9af27..b2acb94 100644
--- a/include/media/ipu-isys.h
+++ b/include/media/ipu-isys.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2014 - 2018 Intel Corporation */
 
 #ifndef MEDIA_IPU_H
diff --git a/include/media/lc898122.h b/include/media/lc898122.h
index b380f47..5c42640 100644
--- a/include/media/lc898122.h
+++ b/include/media/lc898122.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation */
 
 #ifndef __LC898122_HEADER__
diff --git a/include/media/lm3643.h b/include/media/lm3643.h
index 16bb683..58acf6f 100644
--- a/include/media/lm3643.h
+++ b/include/media/lm3643.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef __LM3643_H__
diff --git a/include/media/ti964.h b/include/media/ti964.h
index 8943daf..28ffaac 100644
--- a/include/media/ti964.h
+++ b/include/media/ti964.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation */
 
 #ifndef TI964_H
diff --git a/include/uapi/linux/crlmodule.h b/include/uapi/linux/crlmodule.h
index f0a48f4..8bcb8c9 100644
--- a/include/uapi/linux/crlmodule.h
+++ b/include/uapi/linux/crlmodule.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation */
 
 #ifndef UAPI_LINUX_CRLMODULE_H
diff --git a/include/uapi/linux/ipu-isys-isa-fw.h b/include/uapi/linux/ipu-isys-isa-fw.h
index 0385abc..f0914bc 100644
--- a/include/uapi/linux/ipu-isys-isa-fw.h
+++ b/include/uapi/linux/ipu-isys-isa-fw.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2014 - 2018 Intel Corporation */
 
 #ifndef IPU_ISYS_ISA_FW_H
diff --git a/include/uapi/linux/ipu-isys.h b/include/uapi/linux/ipu-isys.h
index b00e8a7..850767b6 100644
--- a/include/uapi/linux/ipu-isys.h
+++ b/include/uapi/linux/ipu-isys.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2016 - 2018 Intel Corporation */
 
 #ifndef UAPI_LINUX_IPU_ISYS_H
diff --git a/include/uapi/linux/ipu-psys.h b/include/uapi/linux/ipu-psys.h
index f23bc85..0f26dbf 100644
--- a/include/uapi/linux/ipu-psys.h
+++ b/include/uapi/linux/ipu-psys.h
@@ -1,4 +1,4 @@
-/* SPDX-License_Identifier: GPL-2.0 */
+/* SPDX-License-Identifier: GPL-2.0 */
 /* Copyright (C) 2013 - 2018 Intel Corporation */
 
 #ifndef _UAPI_IPU_PSYS_H
-- 
1.9.1

